fix(tabs): added missing direct dependency of action-button in TabsOverflow component (#5429)
adobe/spectrum-web-components: .changeset/tricky-fans-drive.md

---
'@spectrum-web-components/tabs': patch
---

> Added @spectrum-web-components/action-button dependency in Tabs since it uses in the direction button as its direct dependency

Reviewer: ```suggestion
Added `@spectrum-web-components/action-button` as a dependency for Tabs as its used in the direction button.
```

Some word-smithing here and adjusting it to be a complete sentence.

Reviewer 2: @Rajdeepc can we add this in before merge?

> Added @spectrum-web-components/action-button dependency in Tabs since it uses in the direction button as its direct dependency

Reviewer 2: ```suggestion
Added `@spectrum-web-components/action-button` as a dependency for Tabs as it's used in the direction button.
```
Re-adding @castastrophe comment above to commit it

> Added @spectrum-web-components/action-button dependency in Tabs since it uses in the direction button as its direct dependency

Reviewer 2: ```suggestion
Added `@spectrum-web-components/action-button` as a dependency for Tabs as it's used in the direction button.
```
capturing @castastrophe commit suggestion
